src/Core/AdminConsole/Services/Implementations/EventIntegrations/EventIntegrationHandler.cs

```diff
@@ -27,54 +24,35 @@ public async Task HandleEventAsync(EventMessage eventMessage)
             return;
         }
 
-        var configurations = await configurationRepository.GetConfigurationDetailsAsync(
+        var configurations = await configurationDetailsCache.GetOrAddAsync<T>(
             organizationId,
             integrationType,
             eventMessage.Type);
 
         foreach (var configuration in configurations)
         {
-            try
+            // Evaluate filters - if false, then discard and do not process
```

Comment:
❓ Any risk here with us not logging when there's a situation like this? Why would this even happen?